_includes/js/config.md

```diff
@@ -13,7 +13,26 @@ Parse.Config.get().then(function(config) {
   // Something went wrong (e.g. request timed out)
 });
 ```
+## Save a Config
 
+`ParseConfig` can be managed through the SDK when a `Master Key` is provided. You can save new parameters and if you save already existing parameters they will be automatically updated.
```

Comment:
> when a `Master Key` is provided

ie: only in nodejs :)

Reply:
it doesn't works in a browser with a masterkey ?

Reply:
Master key should NEVER be exposed in the browser! and the JS SDK doesn’t support passing a third argument.